MaterialSkin/HTML/material/html/lang/ru.json

```diff
@@ -374,35 +374,35 @@
 "%1 Years":"%1 Годов",
 "1 day":"1 день",
 "%1 days":"%1 дней",
-"Add URL to queue":"",
-"Add favorite, podcast, or preset":"",
-"Adding albums...":"",
-"Adding tracks...":"",
-"Advanced":"",
-"Advanced search":"",
-"Alt+%1":"",
-"Alt+N":"",
-"Appended all tracks to the play queue":"",
-"Appended track to the play queue":"",
-"Clear queue":"",
-"Configure browse modes":"",
-"Ctrl+%1":"",
-"Ctrl+Shift+%1":"",
-"Decrease volume":"",
-"Don't Stop The Music":"",
-"Enable keyboard shortcuts":"",
-"Go back":"",
-"Go to homescreen":"",
-"Increase volume":"",
-"Keyboard shortcuts":"",
-"Loading...":"",
-"Next track":"",
-"Play/pause":"",
-"Previous track":"",
-"Save queue":"",
-"Scroll queue to current track":"",
-"Spacebar":"",
-"Switch to Nth player":"",
-"⌘+%1":"",
-"⌘+Shift+%1":""
+"Add URL to queue":"Добавить URL в очередь",
+"Add favorite, podcast, or preset":"Добавить избранное, подкаст или предустановленные",
+"Adding albums...":"Добавить альбомы",
+"Adding tracks...":"Добавить треки",
+"Advanced":"Расширенный",
+"Advanced search":"Расширенный поиск",
+"Alt+%1":"Alt+%1",
+"Alt+N":"Alt+N",
+"Appended all tracks to the play queue":"Добавить все треки в очередь проигрывания",
+"Appended track to the play queue":"Добавить трек в очередь проигрывания",
+"Clear queue":"Очистить очередь",
+"Configure browse modes":"Настроить режимы просмотра",
+"Ctrl+%1":"Ctrl+%1",
+"Ctrl+Shift+%1":"Ctrl+Shift+%1",
+"Decrease volume":"Уменьшить громкость",
+"Don't Stop The Music":"Режим Играть без остановки",
+"Enable keyboard shortcuts":"Включить сочетания клавиш",
+"Go back":"Назад",
+"Go to homescreen":"Вернутся на главную",
+"Increase volume":"Увеличить громкость",
+"Keyboard shortcuts":"Сочетания клавиш",
+"Loading...":"Загрузука...",
+"Next track":"Следующий трек",
+"Play/pause":"Играть/Пауза",
+"Previous track":"Предыдущий Трек",
+"Save queue":"Сохранить очередь",
+"Scroll queue to current track":"Перейти к текущему треку",
+"Spacebar":"Пробел",
+"Switch to Nth player":"Переключиться в режим Ntl проигрывателя",
+"⌘+%1":"+%1",
```

Comment:
Why have you removed the meta symbol here?

Reply:
mistake